src/autoscaler/scalingengine/scalingengine.go

```diff
@@ -179,7 +180,12 @@ func (s *scalingEngine) ComputeNewInstances(currentInstances int, adjustment str
 			s.logger.Error("failed-to-parse-percentage", err, lager.Data{"adjustment": adjustment})
 			return -1, err
 		}
-		newInstances = int(float64(currentInstances)*(1+percentage/100) + 0.5)
+		if percentage >= 0 {
```

Comment:
I think this is different with what we discussed.  We need to add/delete at least 1 instance, so the logic is 

if newInstances == currentInstances 
     if precentage > 0, then newInstances = currentInstances + 1
     if percentage < 0, then newInstances = currentIntances -1